docs/topics/mta-assessment-module.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+// * topics/mta-assessment-module.adoc
+
+:_content-type: REFERENCE
+[id="mta-assessment-module_{context}"]
+= Assessment Module
+
+The assessment process for applications that are to be migrated in Red Hat Migration Toolkit for Applications (MTA) has been separated from other tool features and enhanced in version 7.0.0.
+
+* A separate new Assessment hub includes the Assessment module, replacing the older _Pathfinder_ feature.
+* Assessment supports multiple questionnaires, relevant to each assessed application.
+* Default questionnaires can be customized and saved.
+* User can design and version custom questionnaires and questions using YAML syntax.
+* Assessed applications are grouped into *archetypes* with similar application characteristics.
+* Assessment module integrates tightly with the application inventory.
+* Assessment questionnaires can be imported and exported.
+* Enhanced questionnaire capabilities:
+** Conditional questions
+** Application auto-tagging according to answers
+** Automated answers from tags in application archetypes
+
+In the MTA user interface, the Assessment Module appears in a separate menu item. Selecting the item shows the currently saved default and custom questionnaires.
+
+:_content-type: PROCEDURE
+[id="mta-assessment-start-questionnaire_{context}"]
+== Opening an assessment questionnaire
+
+The {ProductName} ({ProductShortName}) assesses applications that are to be migrated according to a set of questions relevant to the application, such as dependencies, [more question types]. {ProductShortName} comes with a default questionnaire/s that the user can edit, customize and save as new questionnaires.
+
```

Comment:
Why is [more question types] formatted in bold? is this a note to yourself? Please rewrite "a default questionnaire/s that the user can edit, customize and save as new questionnaires." Does the product come with more than one default questionnaire? Do we say "the user" or "you"? There should be a comma before "and" since customizing and saving are separate operations. Note that feature descriptions. There are a number of other problems with the text of this PR, but i'm going to focus on the visual aspects of it.